Jenkinsfile

```diff
@@ -1,90 +1,72 @@
 #!groovy
 
-// import shared library: https://github.com/cmu-delphi/jenkins-shared-library
+// Import shared lib.
 @Library('jenkins-shared-library') _
 
-pipeline {
+/*
+   Declare variables.
+   - indicator_list should contain all the indicators to handle in the pipeline.
+   - Keep in sync with '.github/workflows/python-ci.yml'.
+   - TODO: Get this list automatically from python-ci.yml at runtime?
```

Comment:
File an issue for this and cite the issue number here

Reply:
https://github.com/cmu-delphi/covidcast-indicators/issues/527 created.